lib/routes/qust/jw.ts

```diff
@@ -0,0 +1,32 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+const baseUrl = 'https://jw.qust.edu.cn/';
+
+export default async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: `${baseUrl}jwtz.htm`,
+    });
+    const $ = load(response.data);
+    const items = $('.winstyle60982 tr').map((_, element) => {
```

Comment:
With the selector of
```suggestion
    const items = $('.winstyle60982 tr a.c60982').map((_, element) => {
```
you don't need https://github.com/DIYgod/RSSHub/blob/9594200f76ba49dcfcbdf6ac99c1d734436ec276/lib/routes/qust/jw.ts#L14
since it will be filtered out at selector level